system/threads.cpp

```diff
@@ -298,6 +344,20 @@ void spawnUtility()
 	utilityWorker->addEventListener("message", callStart);
 }
 
+// This function is called from pthread_exit in musl.
```

Comment:
this comment is inaccurate now right?

Reply:
Correct, I've moved it completely now. Removed